### ISSUE TITLE
Low resolution

### ISSUE THREAD

**Issue by jaredkhan** [OneZoom/OZtree#891]
The tree viewer isn't using the full resolution of the device it's running on, so appears blurry.

Screenshot contrasting the crisp icon of the location button to the fuzzy canvas contents on a device with a `window.devicePixelRatio` of 2:
![BF184C9C-D538-4891-BBF8-659F9350530E_4_5005_c](https://github.com/user-attachments/assets/4bcf6ad8-eae2-4560-89ec-7b2f94db6807)

Screenshot contrasting the crisp text of a text field to the fuzzy canvas contents on a device with a `window.devicePixelRatio` of 3:

![IMG_4566](https://github.com/user-attachments/assets/dcc688e3-970c-43a3-bccc-c1e4e0248034)

**Comment by davidebbo** [comment]
It looks different for me on Chrome on Windows, also with window.devicePixelRatio=2:

![image](https://github.com/user-attachments/assets/30694fea-4cef-486e-96c2-e126f7459f2a)

But yes, it's fuzzier than the Search text.

**Comment by davidebbo** [comment]
It does get more obvious as I zoom in to 200% in Chrome, resulting in window.devicePixelRatio=4:

![image](https://github.com/user-attachments/assets/5dfd337b-535a-41d6-b9d0-4b3233741b12)

**Comment by jaredkhan** [comment]
Yeah that's the different interpolation between browsers  when scaling up the low resolution image to fill the page.

**Comment by lentinj** [comment]
We don't consider window.devicePixelRatio yet, so all this is kinda expected:

https://github.com/OneZoom/OZtree/blob/f90d82ccab3b168fedeada4ee732c8d1641cddb2/OZprivate/rawJS/OZTreeModule/src/controller/controller.js#L59-L65

...it might be a simple case of doing so here, something like https://developer.mozilla.org/en-US/docs/Web/API/Window/devicePixelRatio#correcting_resolution_in_a_canvas.

There may be other side effects though, from muddling CSS / canvas pixels elsewhere..
